application-example.conf

```diff
@@ -89,45 +137,35 @@ kafka {
 
   maxRequestSize: 250000000
 
-}
+  securityProtocol: "SSL"
 
-worker {
+  consumerPropertyFile: ".../consumer-conf/consumer.properties"
 
-  # maximum number of times across all workers that an attempt should be made to process any given document;
-  # when this property is not provided, retries will not be tracked and no limit will be imposed;
-  # this property requires that zookeeper is available at the specified zookeeper.connectString
-  #maxRetries: 2
+  producerPropertyFile: ".../consumer-conf/producer.properties"
 
-  # number of worker threads to start for each pipeline when running lucille in local / single-JVM mode
-  threads: 2
+  documentDeserializer: # TODO
```

Comment:
can use "com.kmwllc.lucille.message.KafkaDocumentDeserializer"
try to mention that this is the default and doesn't need to be set explicitly unless someone wants to use a custom implementation of Deserializer<Document>

Deserializer<Document> is an implementation of org.apache.kafka.common.serialization.Deserializer that can convert a message on a Kafka topic into a Lucille Document